pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -453,7 +454,6 @@ buildStdenv.mkDerivation {
     libGLU
     libevent
     libjpeg
-    libpng
```

Comment:
If `--with-system-png` is conditional, perhaps we'd better keep `libpng` also explicitly conditional?  Configure phases seem to pass, most likely thanks to propagation of dependencies, but I'd think it's better not to rely on that here.

Reply:
Oops, yeah.